internal/handlers/pg/pgdb/indexes_test.go

```diff
@@ -126,7 +126,7 @@ func TestCreateIndexIfNotExists(t *testing.T) {
 			}
 
 			expectedIndexDef += fmt.Sprintf(
-				" INDEX %s ON \"%s\".%s USING btree (%s)",
```

Comment:
This is rather strange way of quoting schema name only, I'm sure it has a reason. Anyway it's a test.